runners/flink/flink_runner.gradle

```diff
@@ -406,3 +406,18 @@ createPipelineOptionsTableTask('Python')
 // Update the pipeline options documentation before running the tests
 test.dependsOn(generatePipelineOptionsTableJava)
 test.dependsOn(generatePipelineOptionsTablePython)
+
+repositories {
+  maven {
+    name 'ext-libraries'
+    url 'https://linkedin.jfrog.io/artifactory/flink-li-custom/'
+  }
+}
+
+dependencies {
```

Comment:
This part doesn't look right to me. Why we are adding the dependencies again here and where is the version defined?

Reply:
Can you please review the latest upload. Re-using the existing conditional statements.